src/quest_manager/templates/quest_manager/category_list.html

```diff
@@ -15,13 +15,15 @@
   <tr>
     <th>Title</th>
     <th>Icon</th>
+    <th># of Quests</th>
```

Comment:
While you're working in here.... can you add an `XP Available` column too?  Can just use the existing `xp_sum()` method

Reply:
![image](https://user-images.githubusercontent.com/105619909/179636365-38f7e10f-6325-4107-afc7-38073f6b3799.png)
XP Available column added